recipes/peptide-shaker/meta.yaml

```diff
@@ -10,6 +10,7 @@ build:
   number: 0
   #noarch: generic
 
+
```

Comment:
I think you will need to undo this line insertion

Reply:
Do extra empty lines modify the linting or the building of a Conda package?

Reply:
I would expect that any modification will make bioconda think its an update and then the linter will complain about a missing bump.